hydpy/core/variabletools.py

```diff
@@ -1136,7 +1136,7 @@ def _finalise_connections(self) -> None:
     def initinfo(self) -> tuple[Union[float, int, bool, pointerutils.Double], bool]:
         """To be overridden."""
 
-    def __call__(self, *args) -> None:
+    def __call__(self, *args, **kwargs) -> None:
```

Comment:
Was this change on purpose?

Reply:
still unaddressed

Reply:
This change seemed to be necessary when we called the super class of class UH in the case, that args and kwargs are used as parameters. As for now, we do not call the super class anymore, I will remove this change again.